third_party/libadblockplus/grit/js_resources.grd

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
copy-paste from `core` - to be fetched from core and reused (see task description above).